client/http/url.c

```diff
@@ -0,0 +1,284 @@
+#include "url.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+static char *str_hosttype[] = { "host ipv4", "host ipv6", "host domain", NULL };
+
+//static char *strndup(const char *str, int n)
```

Comment:
Is there reason to leave this commented function?

Reply:
It's issue from url.c Not sure what is the purpose of this duplicate function yet. So just leave it here till I figure out what is the purpose.